nvfuser/__init__.py

```diff
@@ -378,6 +324,81 @@ def profile(self):
 
         return fp
 
+    def last_repro_script(self) -> str:
+        assert (
+            self.inputs is not None
+        ), "fd.last_repro_script() cannot provide a repro because fd.execute(inputs, save_repro_state=True) was not set!"
```

Comment:
It might be good to include a note in the error message like "fd.last_repro_script() may only be called once after each call to fd.execute()".

Reply:
This is not true anymore as I used fake tensors as @jjsjann123  suggested.